### ISSUE TITLE
InventroyClickEvent shows error in console sometime

### ISSUE THREAD

**Issue by MrMineToons** [WolfyScript/WolfyUtilities#25]
Hello!

Actually, I`m getting those kind of error and I dont know why, maybe a action of some player? That is weird because happens with different people, doesnt looks like an error someone is trying to do.

```
> [03:42:30] [Server thread/INFO]: kelvi51 issued server command: /home casa
> [03:42:35] [Server thread/INFO]: kelvi51 issued server command: /home troca
> [03:43:05] [JDA Gateway-Worker 1/TRACE]: [net.dv8tion.jda.internal.requests.WebSocketClient] <- {"op":1,"d":143}
> [03:43:05] [JDA MainWS-ReadThread/TRACE]: [net.dv8tion.jda.internal.utils.compress.Decompressor] Decompressing data [34, 70, 13, 0, 0, 0, -1, -1]
> [03:43:05] [JDA MainWS-ReadThread/TRACE]: [net.dv8tion.jda.internal.requests.WebSocketClient] Got Heartbeat Ack (OP 11).
> [03:43:11] [Server thread/DEBUG]: kelvi51 is sending move packets too frequently (22 packets since last tick)
> [03:43:46] [JDA Gateway-Worker 1/TRACE]: [net.dv8tion.jda.internal.requests.WebSocketClient] <- {"op":1,"d":143}
> [03:43:46] [JDA MainWS-ReadThread/TRACE]: [net.dv8tion.jda.internal.utils.compress.Decompressor] Decompressing data [34, 70, 13, 0, 0, 0, -1, -1]
> [03:43:46] [JDA MainWS-ReadThread/TRACE]: [net.dv8tion.jda.internal.requests.WebSocketClient] Got Heartbeat Ack (OP 11).
> [03:43:56] [Server thread/INFO]: kelvi51 issued server command: /home casa1
> [03:44:27] [JDA Gateway-Worker 1/TRACE]: [net.dv8tion.jda.internal.requests.WebSocketClient] <- {"op":1,"d":143}
> [03:44:27] [JDA MainWS-ReadThread/TRACE]: [net.dv8tion.jda.internal.utils.compress.Decompressor] Decompressing data [34, 70, 13, 0, 0, 0, -1, -1]
> [03:44:27] [JDA MainWS-ReadThread/TRACE]: [net.dv8tion.jda.internal.requests.WebSocketClient] Got Heartbeat Ack (OP 11).
> [03:44:43] [Server thread/DEBUG]: kelvi51 is sending move packets too frequently (12 packets since last tick)
> [03:44:43] [Server thread/DEBUG]: kelvi51 is sending move packets too frequently (13 packets since last tick)
> [03:44:43] [Server thread/DEBUG]: kelvi51 is sending move packets too frequently (14 packets since last tick)
> [03:44:51] [Server thread/DEBUG]: kelvi51 is sending move packets too frequently (30 packets since last tick)
> [03:45:00] [Server thread/INFO]: kelvi51 issued server command: /craft
> [03:45:08] [JDA Gateway-Worker 1/TRACE]: [net.dv8tion.jda.internal.requests.WebSocketClient] <- {"op":1,"d":143}
> [03:45:09] [JDA MainWS-ReadThread/TRACE]: [net.dv8tion.jda.internal.utils.compress.Decompressor] Decompressing data [34, 70, 13, 0, 0, 0, -1, -1]
> [03:45:09] [JDA MainWS-ReadThread/TRACE]: [net.dv8tion.jda.internal.requests.WebSocketClient] Got Heartbeat Ack (OP 11).
> [03:45:23] [Server thread/ERROR]: Could not pass event InventoryClickEvent to WolfyUtilities v1.6.3.24
> java.lang.NullPointerException: null
> 	at me.wolfyscript.utilities.util.events.ArmorEquipEvent.<init>(ArmorEquipEvent.java:47) ~[?:?]
> 	at me.wolfyscript.utilities.main.listeners.EquipListener.onClickInventory(EquipListener.java:115) ~[?:?]
> 	at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor598.execute(Unknown Source) ~[?:?]
> 	at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69) ~[patched_1.16.5.jar:git-Paper-613]
> 	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[patched_1.16.5.jar:git-Paper-613]
> 	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.16.5.jar:git-Paper-613]
> 	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:607) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.PlayerConnection.a(PlayerConnection.java:2700) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.PacketPlayInWindowClick.a(SourceFile:32) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.PacketPlayInWindowClick.a(SourceFile:10) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.PlayerConnectionUtils.lambda$ensureMainThread$1(PlayerConnectionUtils.java:35) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.TickTask.run(SourceFile:18) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.executeTask(IAsyncTaskHandler.java:136) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.executeNext(IAsyncTaskHandler.java:109) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.MinecraftServer.bb(MinecraftServer.java:1262) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.MinecraftServer.executeNext(MinecraftServer.java:1255) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.awaitTasks(IAsyncTaskHandler.java:119) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.MinecraftServer.sleepForTick(MinecraftServer.java:1216) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:1130) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$a$0(MinecraftServer.java:289) ~[patched_1.16.5.jar:git-Paper-613]
> 	at java.lang.Thread.run(Thread.java:834) [?:?]
> [03:45:27] [Server thread/ERROR]: Could not pass event InventoryClickEvent to WolfyUtilities v1.6.3.24
> java.lang.NullPointerException: null
> 	at me.wolfyscript.utilities.util.events.ArmorEquipEvent.<init>(ArmorEquipEvent.java:47) ~[?:?]
> 	at me.wolfyscript.utilities.main.listeners.EquipListener.onClickInventory(EquipListener.java:115) ~[?:?]
> 	at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor598.execute(Unknown Source) ~[?:?]
> 	at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69) ~[patched_1.16.5.jar:git-Paper-613]
> 	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[patched_1.16.5.jar:git-Paper-613]
> 	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.16.5.jar:git-Paper-613]
> 	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:607) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.PlayerConnection.a(PlayerConnection.java:2700) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.PacketPlayInWindowClick.a(SourceFile:32) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.PacketPlayInWindowClick.a(SourceFile:10) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.PlayerConnectionUtils.lambda$ensureMainThread$1(PlayerConnectionUtils.java:35) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.TickTask.run(SourceFile:18) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.executeTask(IAsyncTaskHandler.java:136) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.executeNext(IAsyncTaskHandler.java:109) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.MinecraftServer.bb(MinecraftServer.java:1262) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.MinecraftServer.executeNext(MinecraftServer.java:1255) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.awaitTasks(IAsyncTaskHandler.java:119) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.MinecraftServer.sleepForTick(MinecraftServer.java:1216) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:1130) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$a$0(MinecraftServer.java:289) ~[patched_1.16.5.jar:git-Paper-613]
> 	at java.lang.Thread.run(Thread.java:834) [?:?]
> [03:45:27] [Server thread/ERROR]: Could not pass event InventoryClickEvent to WolfyUtilities v1.6.3.24
> java.lang.NullPointerException: null
> 	at me.wolfyscript.utilities.util.events.ArmorEquipEvent.<init>(ArmorEquipEvent.java:47) ~[?:?]
> 	at me.wolfyscript.utilities.main.listeners.EquipListener.onClickInventory(EquipListener.java:115) ~[?:?]
> 	at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor598.execute(Unknown Source) ~[?:?]
> 	at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69) ~[patched_1.16.5.jar:git-Paper-613]
> 	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[patched_1.16.5.jar:git-Paper-613]
> 	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.16.5.jar:git-Paper-613]
> 	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:607) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.PlayerConnection.a(PlayerConnection.java:2700) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.PacketPlayInWindowClick.a(SourceFile:32) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.PacketPlayInWindowClick.a(SourceFile:10) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.PlayerConnectionUtils.lambda$ensureMainThread$1(PlayerConnectionUtils.java:35) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.TickTask.run(SourceFile:18) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.executeTask(IAsyncTaskHandler.java:136) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.executeNext(IAsyncTaskHandler.java:109) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.MinecraftServer.bb(MinecraftServer.java:1262) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.MinecraftServer.executeNext(MinecraftServer.java:1255) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.awaitTasks(IAsyncTaskHandler.java:119) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.MinecraftServer.sleepForTick(MinecraftServer.java:1216) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:1130) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$a$0(MinecraftServer.java:289) ~[patched_1.16.5.jar:git-Paper-613]
> 	at java.lang.Thread.run(Thread.java:834) [?:?]
> [03:45:50] [JDA Gateway-Worker 1/TRACE]: [net.dv8tion.jda.internal.requests.WebSocketClient] <- {"op":1,"d":143}
> [03:45:50] [JDA MainWS-ReadThread/TRACE]: [net.dv8tion.jda.internal.utils.compress.Decompressor] Decompressing data [34, 70, 13, 0, 0, 0, -1, -1]
> [03:45:50] [JDA MainWS-ReadThread/TRACE]: [net.dv8tion.jda.internal.requests.WebSocketClient] Got Heartbeat Ack (OP 11).
> [03:46:01] [Server thread/ERROR]: Could not pass event InventoryClickEvent to WolfyUtilities v1.6.3.24
> java.lang.NullPointerException: null
> 	at me.wolfyscript.utilities.util.events.ArmorEquipEvent.<init>(ArmorEquipEvent.java:47) ~[?:?]
> 	at me.wolfyscript.utilities.main.listeners.EquipListener.onClickInventory(EquipListener.java:155) ~[?:?]
> 	at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor598.execute(Unknown Source) ~[?:?]
> 	at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69) ~[patched_1.16.5.jar:git-Paper-613]
> 	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[patched_1.16.5.jar:git-Paper-613]
> 	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.16.5.jar:git-Paper-613]
> 	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:607) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.PlayerConnection.a(PlayerConnection.java:2700) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.PacketPlayInWindowClick.a(SourceFile:32) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.PacketPlayInWindowClick.a(SourceFile:10) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.PlayerConnectionUtils.lambda$ensureMainThread$1(PlayerConnectionUtils.java:35) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.TickTask.run(SourceFile:18) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.executeTask(IAsyncTaskHandler.java:136) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.executeNext(IAsyncTaskHandler.java:109) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.MinecraftServer.bb(MinecraftServer.java:1262) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.MinecraftServer.executeNext(MinecraftServer.java:1255) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.awaitTasks(IAsyncTaskHandler.java:119) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.MinecraftServer.sleepForTick(MinecraftServer.java:1216) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:1130) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$a$0(MinecraftServer.java:289) ~[patched_1.16.5.jar:git-Paper-613]
> 	at java.lang.Thread.run(Thread.java:834) [?:?]
> [03:46:02] [Server thread/ERROR]: Could not pass event InventoryClickEvent to WolfyUtilities v1.6.3.24
> java.lang.NullPointerException: null
> 	at me.wolfyscript.utilities.util.events.ArmorEquipEvent.<init>(ArmorEquipEvent.java:47) ~[?:?]
> 	at me.wolfyscript.utilities.main.listeners.EquipListener.onClickInventory(EquipListener.java:115) ~[?:?]
> 	at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor598.execute(Unknown Source) ~[?:?]
> 	at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69) ~[patched_1.16.5.jar:git-Paper-613]
> 	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[patched_1.16.5.jar:git-Paper-613]
> 	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.16.5.jar:git-Paper-613]
> 	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:607) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.PlayerConnection.a(PlayerConnection.java:2700) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.PacketPlayInWindowClick.a(SourceFile:32) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.PacketPlayInWindowClick.a(SourceFile:10) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.PlayerConnectionUtils.lambda$ensureMainThread$1(PlayerConnectionUtils.java:35) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.TickTask.run(SourceFile:18) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.executeTask(IAsyncTaskHandler.java:136) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.executeNext(IAsyncTaskHandler.java:109) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.MinecraftServer.bb(MinecraftServer.java:1262) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.MinecraftServer.executeNext(MinecraftServer.java:1255) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.awaitTasks(IAsyncTaskHandler.java:119) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.MinecraftServer.sleepForTick(MinecraftServer.java:1216) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:1130) ~[patched_1.16.5.jar:git-Paper-613]
> 	at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$a$0(MinecraftServer.java:289) ~[patched_1.16.5.jar:git-Paper-613]
> 	at java.lang.Thread.run(Thread.java:834) [?:?]
> [03:46:31] [JDA Gateway-Worker 1/TRACE]: [net.dv8tion.jda.internal.requests.WebSocketClient] <- {"op":1,"d":143}
> [03:46:31] [JDA MainWS-ReadThread/TRACE]: [net.dv8tion.jda.internal.utils.compress.Decompressor] Decompressing data [34, 70, 13, 0, 0, 0, -1, -1]
> [03:46:31] [JDA MainWS-ReadThread/TRACE]: [net.dv8tion.jda.internal.requests.WebSocketClient] Got Heartbeat Ack (OP 11).
> [03:46:58] [Netty Epoll Server IO #3/DEBUG]: Enabled auto read
> [03:46:58] [Netty Epoll Server IO #3/DEBUG]: Enabled auto read
> [03:47:12] [JDA Gateway-Worker 1/TRACE]: [net.dv8tion.jda.internal.requests.WebSocketClient] <- {"op":1,"d":143}
> [03:47:12] [JDA MainWS-ReadThread/TRACE]: [net.dv8tion.jda.internal.utils.compress.Decompressor] Decompressing data [34, 70, 13, 0, 0, 0, -1, -1]
> [03:47:12] [JDA MainWS-ReadThread/TRACE]: [net.dv8tion.jda.internal.requests.WebSocketClient] Got Heartbeat Ack (OP 11).
> [03:47:26] [Netty Epoll Server IO #3/DEBUG]: Timeout
> io.netty.handler.timeout.ReadTimeoutException: null
> [03:47:53] [JDA Gateway-Worker 1/TRACE]: [net.dv8tion.jda.internal.requests.WebSocketClient] <- {"op":1,"d":143}
> [03:47:54] [JDA MainWS-ReadThread/TRACE]: [net.dv8tion.jda.internal.utils.compress.Decompressor] Decompressing data [34, 70, 13, 0, 0, 0, -1, -1]
> [03:47:54] [JDA MainWS-ReadThread/TRACE]: [net.dv8tion.jda.internal.requests.WebSocketClient] Got Heartbeat Ack (OP 11).
> [03:48:35] [JDA Gateway-Worker 1/TRACE]: [net.dv8tion.jda.internal.requests.WebSocketClient] <- {"op":1,"d":143}
> [03:48:35] [JDA MainWS-ReadThread/TRACE]: [net.dv8tion.jda.internal.utils.compress.Decompressor] Decompressing data [34, 70, 13, 0, 0, 0, -1, -1]
> [03:48:35] [JDA MainWS-ReadThread/TRACE]: [net.dv8tion.jda.internal.requests.WebSocketClient] Got Heartbeat Ack (OP 11).
> [03:49:16] [JDA Gateway-Worker 1/TRACE]: [net.dv8tion.jda.internal.requests.WebSocketClient] <- {"op":1,"d":143}
> [03:49:16] [JDA MainWS-ReadThread/TRACE]: [net.dv8tion.jda.internal.utils.compress.Decompressor] Decompressing data [34, 70, 13, 0, 0, 0, -1, -1]
> [03:49:16] [JDA MainWS-ReadThread/TRACE]: [net.dv8tion.jda.internal.requests.WebSocketClient] Got Heartbeat Ack (OP 11).
> [03:49:57] [JDA Gateway-Worker 1/TRACE]: [net.dv8tion.jda.internal.requests.WebSocketClient] <- {"op":1,"d":143}
> [03:49:57] [JDA MainWS-ReadThread/TRACE]: [net.dv8tion.jda.internal.utils.compress.Decompressor] Decompressing data [34, 70, 13, 0, 0, 0, -1, -1]
> [03:49:57] [JDA MainWS-ReadThread/TRACE]: [net.dv8tion.jda.internal.requests.WebSocketClient] Got Heartbeat Ack (OP 11).
```

Those errors do not crash or lag my server for now, but it shows sometime. 
I didnt found a way to reproduce it myself yet!

Version of WolfyUtilities:
[11:35:20 INFO]: [WolfyUtilities] Enabling WolfyUtilities v1.6.3.24
[11:35:20 INFO]: [WolfyUtilities] Minecraft version: 1.16.5
PaperMC-604

I will keep paperMC Updated, any news I will post here.
Thanks for attention!

**Comment by WolfyScript** [comment]
ok, I will have a look into it. Seems to be an issue with the ArmorEquipEvent. 
Do you have any custom items that you use as armor?

**Comment by MrMineToons** [comment]
Yoo, thanks for your attention! 

Acttually I have installed CustomCrafting but I had never use or configured the plugin. I just droped and until today didnt touch nothing...

Also have Vivecraft-Spigot-Extensions, but no one is playing with VR on my server has been 2 months or more...

I dont think there is any other customitem that I may have... I'm not sure about AureliumSkills, but either I had never saw any new item in my game with Aurelium.

There is a list of my plugins: 
[14:55:15 INFO]: Plugins (55): AdvancedBan, AureliumSkills, AutomaticBroadcast, BkShop, BotSentry*, ChatManager, Chunky, ClearLag, CoreProtect, CustomCrafting, DeathMessages, DiscordBotAPI*, DiscordSRV, DualWield*, dynmap*, Dynmap-SimpleClans*, EasyWhitelist*, EntityTrackerFixer, Essentials, EssentialsChat, EssentialsSpawn, GrantX, GriefPrevention, Harbor, HolographicDisplays, LevelledMobs, LPX*, LuckPerms, MBanProxyVPN*, MobEggDrop, nLogin, PlaceholderAPI, PremiumVanish, ProtocolLib, React*, SafeTrade, SilkSpawners, SimpleClans, SkinsRestorer, SkQuery, Skript, StackMob, TAB, TigerReports, TreeFeller, TrollBoss, UltimateAutoRestart, UltimateShops, Vault, VillagerOptimiser, Vivecraft-Spigot-Extensions, Vulcan, WolfyUtilities, WorldEdit, WorldGuard
